share/completions/climate.fish

```diff
@@ -0,0 +1,82 @@
+# fish completion for climate
```

Comment:
This comment is useless. You should probably write a short description of the tool with a link to the project man page (or github) instead.